File: 공부내용정리/알고리즘/programmers/Sort/KthNum/KthNum.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];;
        int idx = 0;
        
        for (int[] command : commands) {
            
            int[] new_array = Arrays.copyOfRange(array, command[0]-1, command[1] );
            Arrays.sort(new_array);
            answer[idx++] = new_array[command[2] - 1];
        }
        
        return answer;    
    }
}

File: 공부내용정리/알고리즘/programmers/Sort/KthNum/KthNum.js
function solution(array, commands) {
  const answer = [];
  
  for ( const command of commands) {
      const new_array = array.slice(command[0] - 1, command[1])
          .sort((a , b) => a - b);
      
      answer.push(new_array[command[2] - 1]);
  }
  
  return answer;
}